--- /etc/apt/sources.list.d/pve-enterprise.bak	2026-01-18 10:01:57.574500060 -0800
+++ /etc/apt/sources.list.d/pve-enterprise.sources	2026-01-18 10:02:58.910038788 -0800
@@ -1,5 +1,5 @@
 Types: deb
-URIs: https://enterprise.proxmox.com/debian/pve
+URIs: http://download.proxmox.com/debian/pve
 Suites: trixie
-Components: pve-enterprise
+Components: pve-no-subscription
 Signed-By: /usr/share/keyrings/proxmox-archive-keyring.gpg
